.standard.yml

```diff
@@ -1 +0,0 @@
-ruby_version: 2.6
```

Comment:
I didnt get the idea of the file itself — standard gem supposed to be unconfigurable if there is no need, so I've decided to remove the file, because didn't see any need in it :)

Hope you don't mind, but I may return it back if you insist. I may also run the autocorrect command as well, linter found 4 little issues

Reply:
This configuration is needed to specify the minimum version of Ruby to accidentally not to use new syntax from Ruby 3 on older Ruby

Reply:
Good point.

I got the file back and also fixed the problems, standard have given, minor ones. Also I've added standard to Github Actions.